test/Microsoft.AspNetCore.Server.IISIntegration.Tests/IISMiddlewareTests.cs

```diff
@@ -191,7 +201,7 @@ public void PathBaseHiddenFromServer()
                 {
                     app.Run(context =>
                     {
-                        var auth = context.Features.Get<IHttpAuthenticationFeature>();
+                        var auth = context.RequestServices.GetService<IAuthenticationSchemeProvider>();
```

Comment:
Updated test to verify handler only added when forward = true, also added a variant that ensures things don't blow up without auth services.